problems/objects.js

```diff
@@ -151,15 +203,30 @@ const authorScores = () => {};
  * @returns {string} The full name of person with best score.
  */
 
-const bestScore = () => {};
+const bestScore = (submissions) => {
+  for(let i = 0; i < submissions.length; i++){
+    if(submissions[i]["score"] > submissions[submissions.length - 1]["score"]){
+      return submissions[i]["firstName"] +  " " + submissions[i]["lastName"]  
```

Comment:
This working is probably a coincidence. We don't want to return in the if statement, instead we want to reassign the value. Returning would end the function and we wouldn't get to check any other scores.